Assets/Scripts/Controllers/SceneNavigator.cs

```diff
@@ -51,7 +46,7 @@ public void changeSceneIfSatisfied(Asset clickedAsset)
 
     private void showErrorScene(Asset clickedAsset, Conditional failedCondition)
     {
-        Scene errorToShow = _errorSceneTable[(clickedAsset, failedCondition)];
+        Cutscene errorToShow = _errorSceneTable[(clickedAsset, failedCondition)];
 
         _currentScene.hide();
         setCurrentScene(errorToShow);
```

Comment:
Given that `showErrorScene` and `showNextScene` are very similar but with one line differences, the two methods can be merged into a method called `showScene`, as follows:
```csharp
private void showScene(Scene newScene)
{
    _currentScene.hide();
    setCurrentScene(newScene);
    newScene.show();
}
```

turning lines 39-40 and 44-45 to something like
```csharp
if (!condition.isMet())
{
    Cutscene errorToShow = _errorSceneTable[(clickedAsset, failedCondition)];
    showScene(errorToShow);
    return;
}

...
showNextScene(clickedAsset.getState().getNextScene());
```

Thoughts?

Reply:
Oh, I like that idea.

Reply:
I've included this change in the most recent commit.